astroquery/casda/core.py

```diff
@@ -123,6 +141,178 @@ def _parse_result(self, response, verbose=False):
             print("Failed to convert query result to table", e)
             return response
 
+    def filter_out_unreleased(self, table):
+        """
+        Return a subset of the table which only includes released (public) data.
+        :param table: A table of results as returned by query_region. Must include an obs_release_date column.
```

Comment:
Could you please format these in numpy doc format?   Have a look at other examples through astropy/astroquery to see how that's done.  e.g.:
```
Parameters
---------
    table : `astropy.table.Table`
       <description>
```

Reply:
Thanks, I've corrected the format of the docstrings.